ethclient/ethclient.go

```diff
@@ -505,6 +505,46 @@ func (ec *Client) SuggestGasTipCap(ctx context.Context) (*big.Int, error) {
 	return (*big.Int)(&hex), nil
 }
 
+// FeeHistoryResult is the result of FeeHistory.
+type FeeHistoryResult struct {
```

Comment:
I do wonder if this is the right approach. It seems like it might be more correct to include in `interfaces.go` since right now the only thing exported by `ethclient` is `Client`?

Reply:
@fjl updated in https://github.com/ethereum/go-ethereum/commit/5ce54d31106469b8149ead8738b653656b5593f7, PTAL when you have a moment!